stoplight.js

```diff
@@ -0,0 +1,20 @@
+$("#red").on("click",changeToRed);
+  function changeToRed() {
+   $("#red").css("background","red");
+  }
+$("#yellow").on("click",changeToYellow);
+    function changeToYellow() {
+     $("#yellow").css("background","yellow");
+    }
+$("#green").on("click",changeToGreen);
+      function changeToGreen() {
+       $("#green").css("background","green");
+      }
+      // $("#red").on("click",changeToBlack);
+      //   function changeToBlack() {
+      //    $("#red").css("background","black");
+      //  }
+       $('#red').on("click",changeToBlac);
+       function changeToBlac(){}
+       $(this).css("background","black");
+     }
```

Comment:
- How could you turn a light off?
- How could you make it so only one color can be on at a time?
- When it turns yellow, how could you make it wait 2 seconds before turning red?